util/get_data_from_secret.go

```diff
@@ -0,0 +1,31 @@
+package util
+
+import (
```

Comment:
would it make sense to rename the file to "secret.go" rather than get_data_from_secret.go ? You can add additional utility functions under this file.

Reply:
Makes sense!